kube/src/config/file_config.rs

```diff
@@ -38,7 +38,7 @@ pub struct Preferences {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct NamedExtension {
     pub name: String,
-    pub extension: String,
+    pub extension: serde_json::Value,
```

Comment:
Are there specs for this anywhere? It's probably fine to have that as dynamic, though it is technically a breaking change for previous String extensions.

Reply:
Ah, I see it under https://github.com/kubernetes/client-go/blob/30548acd0a9e231d42d406433aef55b111b36240/tools/clientcmd/api/v1/types.go#L178-L184 .

Reply:
Interestingly it is a runtime.RawExtension from https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#rawextension

Reply:
I haven't found much in the way of `RawExtension` specs beyond the links you've listed above and some of the references in this thread: https://github.com/kubernetes/apimachinery/issues/102 and the recent PR for `minikube` that began using the `extensions`: https://github.com/kubernetes/minikube/pull/10126 (which seems to have required a similar change to this one in the C# client: https://github.com/kubernetes-client/csharp/pull/556)